store/s3store.go

```diff
@@ -71,16 +74,14 @@ func NewS3Store(numRetries int) *S3Store {
 
 	bucket, ok := os.LookupEnv(BucketEnvVar)
```

Comment:
Unrelated, but I find non-main packages looking at the env to be very distasteful :P